recordio/file_reader_test.go

```diff
@@ -187,6 +189,11 @@ func TestReaderForbidsDoubleOpens(t *testing.T) {
 	expectErrorStringOnOpen(t, reader, "already opened")
 }
 
+func TestReaderInitNoPath(t *testing.T) {
+	_, err := NewFileReader()
+	assert.Equal(t, errors.New("NewFileReader: either os.File or string path must be supplied, never both"), err)
+}
```

Comment:
sorry to be pedantic :rofl: but can you add another where to supply both?

Reply:
Done